api/service.py

```
@@ -545,15 +675,15 @@ async def _process_telemetry(telemetry: TelemetryInput, request_start: float) ->
 
 
 @app.get("/api/v1/telemetry/latest")
-async def get_latest_telemetry():
+async def get_latest_telemetry(api_key: APIKey = Depends(get_api_key)):
```

Ruff (0.14.10)

Unused function argument: `api_key`

(ARG001)